gui/wxpython/animation/dialogs.py

```diff
@@ -1560,6 +1573,59 @@ def _hideAll(self):
         self.hidevbox.Show(self.informBox, True)
         self.hidevbox.Layout()
 
+    def _export_file_validation(self, filebrowsebtn, file_path,
+                                file_postfix):
+        """File validation before export
+
+        :param obj filebrowsebutton: filebrowsebutton widget
+        :param str file_path: exported file path
+        :param str file_postfix: exported file postfix
+        (.gif, .swf, .avi)
+
+        :return bool: True if validation is ok
+        """
+
+        file_path_does_not_exist_err_message = _(
+            "Exported file directory '{base_dir}' "
+            "does not exist."
+        )
+        if not file_path:
+            GError(parent=self, message=_("Export file is missing."))
+            return False
+        else:
+            if file_postfix not in file_path:
```

Comment:
This should probably use endswith(), I can imagine users naming their files with e.g. 'gif' in the name.

Reply:
> This should probably use endswith(), I can imagine users naming their files with e.g. 'gif' in the name.

Yes of course.